server/graphql/v2/object/Host.js

```diff
@@ -265,6 +266,16 @@ export const Host = new GraphQLObjectType({
             merchantId = (await fetchAccountWithReference(args.merchantAccount, { throwIfMissing: true })).id;
           }
 
+          let collectiveIds;
```

Comment:
:ice_cream: Personally, I'm not a big fan of mutability and I think it is clear to add the conditional when declaring the constant.
```js
const collectiveIds = isEmpty(args.collectiveAccounts) ? undefined : await Promise.all(...).then(collectives => collectives.map(c => c.id))
```

Reply:
Done. 👍🏽